ftl/node/main.py

```diff
@@ -57,6 +57,12 @@
     help='The path where the application data sits.')
 
 parser.add_argument(
+    '--no-cache',
```

Comment:
double negatives can be a bit hard to reason about, it might make sense to name this flag --cache and default it to true. Then we don't need "if not no_cache" above.

Reply:
so then when --cache is passed in, that actually means not to use the cache? would it make more sense to keep it as --no-cache, and pass the negative in to the builder?

Reply:
Sorry, to be clear --cache would be a boolean that defaults to true. So you would do --cache=False to turn it off.